src/dialogs/Dialogs/OldVersionDialog.tsx

```diff
@@ -0,0 +1,33 @@
+import Button, { ButtonColor } from "@app/components/Button";
+import { BaseDialog } from "./BaseDialog";
+import { closeDialog } from "..";
+
+export class OldVersionDialog extends BaseDialog<Record<string, never>> {
+    constructor(props: Record<string, unknown>) {
+        super(props);
+    }
+
+    getInnerContents() {
+        return <>
+            <p>
+                <strong>You are trying to select an old version of YARG!</strong>
```

Comment:
Make consistent with dialog title
```suggestion
                <strong>You are trying to select an old version of this application!</strong>
```